usr.sbin/daemon/daemon.c

```diff
@@ -65,6 +65,8 @@ enum daemon_mode {
 };
 
 struct daemon_state {
+	long restart_delay;
```

Comment:
I don't think I'd bother with widening these, to be honest; `restart_delay` is in seconds and INT_MAX is a lot of seconds to wait, probably not reasonable. `stdmask` is 0, 1, 2, or 1 | 2.

IMO w should instead would convert those two `strtol` calls to `strtonum` with a range of [0, INT_MAX], cast the result to (int) and call it a day.